File: 240805/ab 사이에 없는 c/c-between-a-and-b-2.py
arr=input().split()
a=int(arr[0])
b=int(arr[1])
c=int(arr[2])
satisfied = True

for i in range(a,b+1):
    if i % c == 0:
        satisfied = False

if satisfied == True:
    print("YES")
else:
    print("NO")

File: 240830/ab 사이에 없는 c/c-between-a-and-b-2.cpp
#include <iostream>
using namespace std;

int main() {
    
    int a, b, c;
    cin >> a >> b >> c;

    bool satisfied = true;
    for (int i = a; i <= b; i++) {
        if (i % c != 0) {
            satisfied = false;
        }
    }
    if (satisfied == true) {
        cout << "NO";
    }
    else {
        cout << "YES";
    }
    return 0;
}